lib/projects/ProjectLogsManager.ts

```diff
@@ -133,12 +146,11 @@ class ProjectLogsManager {
 
     if (this.selectedFunction.deployOutput.endpoint) {
       this.endpointName = this.selectedFunction.deployOutput.endpoint.path;
-      this.method = this.selectedFunction.deployOutput.endpoint.method;
```

Comment:
This was both trying to access a field that doesn't exist (it should be `methods`) _and_ not actually used anywhere so I just removed it

Reply:
😭 Well, it's good we caught it now